cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Given(/^I save the amount of log lines into "([^"]*)"$/) do |bookmark|
+  save_amount_of_log_lines(bookmark)
+end
+
+
+And(/^The log filtered from line number in "([^"]*)" should contains "([^"]*)" messages:$/) do |bookmark, occurrences, message|
```

Comment:
"The log filtered from bookmark "([^"]*)" contains "([^"]*)" messages:"
i think this is better